snuba/datasets/configuration/events/storages/errors_ro.yaml

```diff
@@ -264,8 +264,8 @@ allocation_policies:
         is_active: 1
       cross_org_referrer_limits:
         getsentry.tasks.backfill_grouping_records:
-          max_threads: 2
-          concurrent_limit: 4
+          max_threads: 10
```

Comment:
if you're doing point queries you don't need this to be this high

Reply:
👍🏼